openvdb/tools/MeshToVolume.h

```diff
@@ -825,11 +826,13 @@ class SweepExteriorSign
 
     void operator()(const tbb::blocked_range<size_t>& range) const {
 
+        constexpr Int32 DIM = static_cast<Int32>(LeafNodeType::DIM);
```

Comment:
Re-worked this method to use signed arithmetic for `pos` and `step` variables as `step` can decrement